### ISSUE TITLE
npm audit fail for package minimist

### ISSUE THREAD

**Issue by DanielRyan** [tonyganch/gonzales-pe#306]
Running `npm audit` / `npm audit --fix` fails for package `minimist`.
https://www.npmjs.com/advisories/1179

```

                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Low             Prototype Pollution

  Package         minimist

  Patched in      >=0.2.1 <1.0.0 || >=1.2.3

  Dependency of   stylelint [dev]

  Path            stylelint > postcss-sass > gonzales-pe > minimist

  More info       https://npmjs.com/advisories/1179
```

**Comment by Hypnosphi** [comment]
It's actually fixed in master, but still unreleased
https://github.com/tonyganch/gonzales-pe/issues/302

**Comment by fabb** [comment]
@tonyganch please release a new version

**Comment by fabb** [comment]
Since the repo owner is not responding, maybe @jdalton could you release a new version from `dev`?

**Comment by tonyganch** [comment]
Published as v4.3.0 🐙

**Comment by fabb** [comment]
Thank you! ❤️
